.github/workflows/early-access.yaml

```diff
@@ -47,7 +44,7 @@ jobs:
           distribution: 'temurin'
 
       - name: 'Run default (non-native) build'
-        run: ./mvnw verify -Dmrm=false -V -B -ntp -e -s .mvn/release-settings.xml
+        run: ./mvnw verify site -Dmrm=false -V -B -ntp -e -s .mvn/release-settings.xml
```

Comment:
I'd not interleave site into this one... @gnodet ?

Reply:
Yes, maybe another job would be better specifically for the site.

Reply:
Moved it to another Job